### PR TITLE
Frontend: Extract CSS imports into files

### PR DESCRIPTION
**Pull request by jackw**
**What is this feature?**

This PR makes changes to the webpack css loaders config, the backend webassets, and the html templates so CSS imports within the frontend codebase are extracted into separate files and loaded as .css files rather than [bundled inside the JS](https://grafana-assets.grafana.net/grafana/11.3.0-77222/public/build/app.86953ada7cc7df706145.js).

### before
Screenshot 2024-10-12 at 17 30 26

### after
image

**Why do we need this feature?**

So we load css as css files and not bundle it in our javascript files.

**Who is this feature for?**

Everybody. 🚀 

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

I could do with a bit of help testing this as it should load the entrypoint and then later webpack should async load any css chunks that accompany js chunks.

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: pkg/api/webassets/webassets_test.go [hunk 1]
```diff
@@ -15,32 +15,48 @@ func TestReadWebassets(t *testing.T) {
 
 	dto, err := json.MarshalIndent(assets, "", "  ")
 	require.NoError(t, err)
-	// fmt.Printf("%s\n", string(dto))
+	fmt.Printf("%s\n", string(dto))
```
Comment (marefr) [inline pkg/api/webassets/webassets_test.go:18]:
```suggestion
	// fmt.Printf("%s\n", string(dto))
```

**Review by leventebalogh — APPROVED** [review]

**Review by marefr — APPROVED** [review]
Backend LGTM

**Pull request merged**